DiscordPlayerCountBot/DiscordClient.cs

```diff
@@ -25,10 +53,15 @@ public static async Task SetChannelName(this IDiscordClient socket, ulong? chann
 
             if (channel != null)
             {
-                if (channel is ITextChannel)
+                if (channel is IVoiceChannel)
```

Comment:
I believe you are looking to only update the channel if it's a text channel, and not when it's a voice channel.

```csharp
if (channel is ITextChannel && channel is not IVoiceChannel)
{
    gameStatus = gameStatus.Replace('/', '-').Replace(' ', '-').Replace(':', '-');
}
```